ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -390,6 +394,11 @@ tests:
     env:
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-gcp
+- as: e2e-aws-ovn-kubevirt
+  optional: true
```

Comment:
as a starting point, what about adding `run_if_changed: ^test/extended/kubevirt/.*` so this only runs optionally when our kubevirt tests are modified?